packages/animation-curve-block/src/components/Card.tsx

```diff
@@ -0,0 +1,238 @@
+                    toolbarFlyoutItems={[]}
+                >
+                    <div
+                        data-test-id="animation-curve-card"
+                        onMouseEnter={!isEditFlyoutOpen ? () => setIsHovered(true) : undefined}
+                        onMouseLeave={!isEditFlyoutOpen ? () => setIsHovered(false) : undefined}
+                        className={joinClassNames([
+                            (hasBackground || hasBorder) && 'tw-overflow-hidden',
+                            'tw-flex tw-flex-col tw-relative tw-bg-base',
+                        ])}
+                        style={{
+                            ...(hasBorder && getBorderStyles(borderStyle, borderWidth, borderColor)),
+                            ...getRadiusStyles(radiusChoice, hasRadius, radiusValue),
+                        }}
+                    >
+                        <AnimationCurveFlyout
+                            animationCurve={localAnimationCurve}
+                            isFlyoutOpen={isEditFlyoutOpen}
+                            lineColor={lineColor}
+                            endpointColor={endpointsColor}
+                            gridColor={gridColor}
+                            onSave={(id) => {
+                                onUpdate(id, localAnimationCurve);
+                                setIsEditFlyoutOpen(false);
+                                setIsHovered(false);
+                            }}
+                            onCancel={() => {
+                                setIsEditFlyoutOpen(false);
+                                setIsHovered(false);
+                            }}
+                            onOpenChange={(isOpen) => {
+                                setIsEditFlyoutOpen(isOpen);
+                                if (!isOpen) {
+                                    setIsHovered(false);
+                                }
+                            }}
+                            onAnimationCurveChange={setLocalAnimationCurve}
+                        />
+                        <div
+                            data-test-id="animation-curves-canvas-wrapper"
+                            className={merge([(hasBorder || hasBackground) && 'tw-px-5 tw-py-4'])}
+                            style={{
+                                ...(hasBackground && getBackgroundColorStyles(backgroundColor)),
+                            }}
+                        >
+                            <AnimationCanvas
+                                animationFunction={animationFunction}
+                                showGrid={hasGrid}
+                                showEndPoints={hasEndpoints}
+                                lineColor={lineColor}
+                                endpointColor={endpointsColor}
+                                gridColor={gridColor}
+                                viewBox={{
+                                    width: 350,
+                                    height: 200,
```

Comment:
define in a const?

Reply:
defined a constant `DEFAULT_ANIMATION_CANVAS_VIEWBOX`